### PR TITLE
Create empty projects for contracts module

### PR DESCRIPTION
**Pull request by kamilbaczek**
Pull Request includes:
- new empty projects for contracts module
- restructure virtual solution structure 
Pull Request not includes:
- reorganize physical dictionaries structure
- moved classes from directory to projects

### CODE REVIEW

File: Chapter-2-modules-separation/Src/Fitnet.sln [hunk 1]
```diff
@@ -6,6 +6,16 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Fitnet.IntegrationTests", "
...
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Fitnet.Contracts.Api", "Fitnet.Contracts.Api\Fitnet.Contracts.Api.csproj", "{E0941386-F778-450A-9E8A-A282E6ACDD4B}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Fitnet.Contracts.Infrastructure", "Fitnet.Contracts.Infrastructure\Fitnet.Contracts.Infrastructure.csproj", "{A1E1A7EE-DAC4-470D-A778-FC177777656D}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Fitnet.Contracts.Core", "Fitnet.Contracts.Core\Fitnet.Contracts.Core.csproj", "{0DD647FA-2AD7-4C94-B51A-C6AFCDE20C40}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Src", "Src", "{F97E83AB-6F3B-4E23-A7E7-3EB42A8AF20B}"
+EndProject
 Global
```
Comment (meaboutsoftware) [inline Chapter-2-modules-separation/Src/Fitnet.sln:19]: I am missing Contracts.IntegrationTests and Contracts.Tests :)

File: Chapter-2-modules-separation/Src/Fitnet.Contracts.Api/Fitnet.Contracts.Api.csproj [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
```
Comment (meaboutsoftware) [inline Chapter-2-modules-separation/Src/Fitnet.Contracts.Api/Fitnet.Contracts.Api.csproj:4]: Same as in my case, drop the TargetFramework (this node is created as default) :)

**Review by meaboutsoftware — CHANGES_REQUESTED** [review]

File: Chapter-2-modules-separation/Src/Fitnet.Contracts.Api/Fitnet.Contracts.Api.csproj (hunk 2 above)
Comment (kamilbaczek) [reply Chapter-2-modules-separation/Src/Fitnet.Contracts.Api/Fitnet.Contracts.Api.csproj:4]: Yep

File: Chapter-2-modules-separation/Src/Fitnet.Contracts.Api/Class1.cs [hunk 3]
```diff
@@ -0,0 +1,5 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.Contracts.Api;
```
[the remark this answers is not in the document]
Comment (kamilbaczek) [reply Chapter-2-modules-separation/Src/Fitnet.Contracts.Api/Class1.cs]: Done

File: Chapter-2-modules-separation/Src/Fitnet.sln (hunk 1 above)
Comment (kamilbaczek) [reply Chapter-2-modules-separation/Src/Fitnet.sln:23]: I'm gonna add it in next PR.

File: Chapter-2-modules-separation/Src/Fitnet.sln [hunk 4]
```diff
@@ -12,6 +12,14 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Fitnet.Reports.IntegrationT
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Reports", "Reports", "{7D3193F0-C731-4683-893F-53CE8087D7E2}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Contracts", "Contracts", "{1DD15708-BB12-49FA-A259-F8BC147FD57F}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Fitnet.Contracts.Api", "Fitnet.Contracts.Api\Fitnet.Contracts.Api.csproj", "{B070E263-F998-4086-B16B-BD467652C219}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Fitnet.Contracts.Application", "Fitnet.Contracts.Application\Fitnet.Contracts.Application.csproj", "{C7048CE6-7C8E-4C73-88A7-7119485ADFA8}"
```
Comment (meaboutsoftware) [inline Chapter-2-modules-separation/Src/Fitnet.sln:19]: Now you lost Infrastructure - it is not in solution:

image

**Review by meaboutsoftware — CHANGES_REQUESTED** [review]

File: Chapter-2-modules-separation/Src/Fitnet.sln (hunk 4 above)
Comment (kamilbaczek) [reply Chapter-2-modules-separation/Src/Fitnet.sln:19]: you are right, sorry :D Fixed

**Review by meaboutsoftware — APPROVED** [review]

**Pull request merged**